packages/eslint-config-qlean-react-native/README.md

```diff
@@ -0,0 +1,53 @@
+# eslint-config-qlean-react-native
+
+This package provides Qlean's `.eslintrc` as an extensible shared config for linting React Native codebase.
+
+## Installation
+
+1. Install the correct versions of each package, which are listed by the command:
+```sh
+npm info "eslint-config-qlean-react-native" peerDependencies
+```
+2. Install the latest version of `eslint-config-qlean-react-native` package with following command:
+```sh
+npm install --save-dev eslint-config-qlean-react-native
+```
```

Comment:
Comments are redundant, `--save-dev` too. Just write one actual command, like:
`$ npm install eslint-config-qlean-react-native and-all-peer-deps`

Reply:
@alextewpin Why `--save-dev` is redundant?

Reply:
My bad, `--save` is.